bin/i18n/sync-codeorg-in

```diff
@@ -57,19 +57,20 @@ def localize_level_content
   level_authored_hints = Hash.new
   level_callouts = Hash.new
 
+  instruction_pattern = /^\s*"instructions": (".*?"),?\n$/
+  markdown_instruction_pattern = /^\s*"markdown_instructions": (".*?"),?\n$/
+  failure_message_override_pattern = /^\s*"failure_message_override": (".*?"),?\n$/
+  authored_hint_pattern = /^\s*"authored_hints": "(.*?)",?\n$/
+  callout_pattern = /^\s*"callout_json": "(.*?)",?\n$/
+
   Dir.glob("dashboard/config/scripts/levels/*.level").each do |file|
     level = File.basename(file, ".*") + "_instruction"
+    failure_message_override_level = File.basename(file, ".*") + "_failure_message_override"
```

Comment:
Nit: Can we move this to line 69 so it keeps the same pattern as the rest of the script?